actions/sync-calendar/gcal/testing/test_schedule.yaml

```diff
@@ -0,0 +1,11 @@
+# Generated from: go run actions/generate-schedule/main.go --start 2020-06-01 --stop 2020-07-01 --users abc,lmn,xyz --out actions/sync-calendar/gcal/testing/test_schedule.yaml
+shifts:
+- startDate: Mon 01 Jun 2020
```

Comment:
Should we override a user here to test that code path?

Reply:
Whoops - I know you made that comment in the last review and it didn't get ported over. Will add that.